Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Rundll32.exe_executes_a_rare_unsigned_module.yml

```diff
@@ -0,0 +1,1531 @@
+      todate:
+        simple: now
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1184,
+          "y": 914
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "36":
+    id: "36"
+    taskid: d292b514-4f32-4934-8800-19a3e352c65f
+    type: regular
+    task:
+      id: d292b514-4f32-4934-8800-19a3e352c65f
+      version: -1
+      name: Extract the DLL's extension
+      description: Extract the DLL's extension.
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      append:
+        simple: "true"
+      key:
+        simple: dll_extension
+      value:
+        complex:
+          root: Core.OriginalAlert.event
+          accessor: action_module_name
+          transformers:
+          - operator: RegexExtractAll
+            args:
+              error_if_no_match: {}
+              ignore_case: {}
+              multi_line: {}
+              period_matches_newline: {}
+              regex:
+                value:
+                  simple: \.(\w+)$
+              unpack_matches: {}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1050,
+          "y": -246
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "37":
+    id: "37"
+    taskid: d3e8c9d9-2857-497e-8db9-09cd9ee5d6ee
+    type: condition
+    task:
+      id: d3e8c9d9-2857-497e-8db9-09cd9ee5d6ee
+      version: -1
+      name: Check if the fuzzyhash is not similar
+      description: Remove False Positives in case the CGO signer is similar to the Company name in the DLL's attribute (excluding Microsoft Corporation).
```

Comment:
Remove false positives if the CGO signer is similar to the company name in the DLL's attribute (excluding Microsoft Corporation).

Reply:
Done